src/toil/batchSystems/kubernetes.py

```diff
@@ -803,6 +809,26 @@ def _check_accelerator_request(self, requirer: Requirer) -> None:
                     ],
                 )
 
+    class FakeResponse:
```

Comment:
It might be nice to have some documentation about how this is part of the hack for deserializing. Maybe we could put the class in the function actually without causing problems? We aren't really letting the objects of the class escape, and it would be more understandable.